src/fides/data/sample_project/docker-compose.yml

```diff
@@ -61,11 +61,7 @@ services:
     volumes:
       - type: bind
         source: ./privacy_center/config
-        target: /fides/clients/privacy-center/config
-        read_only: False
-      - type: bind
-        source: ./privacy_center/public/assets
-        target: /app/public/assets
```

Comment:
NOTE: this is a breaking change I'll need to document. I didn't realize this, but the `public/` folder of a NextJS app is only usable if you put everything you want to host into it at *build-time*. You can't actually add files at runtime and expect NextJS to do static file hosting, etc.

Interesting decision but does mean that things like logos/etc. should be hosted elsewhere. We'll need to document this change and see if it hurts anyone's usage